src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[fromFormat.length - 1];
+  const separatorNew = toFormat[toFormat.length - 1];
+
+  const arrDate = date.split(separator);
+
+  const newDate = {};
+
+  for (let i = 0; i < arrDate.length; i++) {
+    newDate[fromFormat[i]] = arrDate[i];
+  }
+
+  if ('YYYY' in newDate) {
+    newDate.YY = newDate['YYYY'].slice(-2);
+  } else if (newDate['YY'] < 30) {
```

Comment:
```suggestion
  } else if (newDate.YY < 30) {
```
fix all similar cases